wscript

```diff
@@ -440,7 +431,7 @@ def configure(cfg):
     cfg.env.BOARD = cfg.options.board
     cfg.env.DEBUG = cfg.options.debug
     cfg.env.COVERAGE = cfg.options.coverage
-    cfg.env.FORCE32BIT = cfg.options.force_32bit
+    cfg.env.SITL32BIT = cfg.options.sitl_32bit
```

Comment:
please double check where and why these changes are coming from, they shouldn't be here I think

Reply:
cleaned up. marked as outdated now